website/assets/css/style.css

```
@@ -1,3 +1,19 @@
 body{
-    background: #BDBDBD;
-}
\ No newline at end of file
+    <<<<<<< tagline
+        background: #BDBDBD;
+    }
+    
+    .bg-image {
+        height: 400px;
+    =======
+        background: white;
+    }
+    
+    .container-fluid{
+        padding: 0;
+    }
+    
+    footer{
+        margin: 0;
+    >>>>>>> master
```

You commited the merge conflict changes, you should never do this since it breaks the code, leave as before the merge as below:

```suggestion
        background: #BDBDBD;
    }
    
    .bg-image {
        height: 400px;
```

---

acabo de resolver esto